inc/saiswitch.h

```diff
@@ -610,6 +610,30 @@ typedef enum _sai_switch_hostif_oper_status_update_mode_t
 
 } sai_switch_hostif_oper_status_update_mode_t;
 
+/**
+ * @brief Attribute data for SAI_SWITCH_ATTR_HOSTIF_OPER_STATUS_UPDATE_MODE.
+ */
+typedef enum _sai_packet_trimming_queue_resolution_mode_t
+{
+    /**
+     * @brief Static queue resolution.
+     *
+     * In this mode, a new queue for the trimmed packet is set directly
+     * by the application.
+     */
+    SAI_PACKET_TRIMMING_QUEUE_RESOLUTION_MODE_STATIC,
+
+    /**
+     * @brief Dynamic queue resolution.
+     *
+     * In this mode, a new queue for the trimmed packet is resolved
```

Comment:
In case of dynamic mode, DSCP-->TC-->Queue should be based on ingress port's packet path. Please specify this order.